pkg/dashboard/server/gitopsManifests.go

```diff
@@ -0,0 +1,83 @@
+package server
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/gimlet-io/gimlet-cli/pkg/dashboard/store"
+	"github.com/gimlet-io/gimlet-cli/pkg/git/nativeGit"
+	"github.com/go-chi/chi"
+	"github.com/sirupsen/logrus"
+)
+
+func getGitopsManifests(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+	store := ctx.Value("store").(*store.Store)
+	gitRepoCache, _ := ctx.Value("gitRepoCache").(*nativeGit.RepoCache)
+
+	envName := chi.URLParam(r, "env")
+	environment, err := store.GetEnvironment(envName)
+	if err != nil {
+		logrus.Errorf("cannot get env from db: %s", err)
+		http.Error(w, http.StatusText(http.StatusInternalServerError), http.StatusInternalServerError)
+		return
+	}
+
+	path := filepath.Join(environment.Name, "flux")
+	if environment.RepoPerEnv {
+		path = "flux"
+	}
+
+	infraRepoFiles, err := gitopsManifests(gitRepoCache, environment.InfraRepo, path)
+	if err != nil {
+		logrus.Errorf("cannot get gitops manifests from repo: %s", err)
+		http.Error(w, http.StatusText(http.StatusInternalServerError), http.StatusInternalServerError)
+		return
+	}
+
+	appsRepoFiles, err := gitopsManifests(gitRepoCache, environment.AppsRepo, path)
```

Comment:
Please remove the apps repo. We don't need to apply in env connect.

Infra repo has a Flux kustomization that points to it, so it will be applied eventually